worlds/lingo/static_logic.py

```diff
@@ -0,0 +1,512 @@
+import yaml
```

Comment:
It'd be nice if this file had comments, even just to break up the procedure into sections. It's hard to stay focused on what's being accomplished at any given moment even though you're pretty much just iterating through the yaml.